framework/src/application/node/processor/processor.ts

```diff
@@ -441,6 +441,7 @@ export class Processor {
 			updateDelegates: async (delegates: Delegate[]): Promise<void> => {
 				await this._chain.setValidators(delegates, stateStore);
 			},
+			getDelegates: async (): Promise<Delegate[]> => this._chain.getValidators(),
```

Comment:
getBootStrapLastHeight is missing implementation, which can be implemented in `chain` and used here